spec/menu_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'menu'
+
+describe Menu do
+  before do
+    @burger = double("Burger", :name => "Burger", :price => 8.5)
```

Comment:
Good use of doubling. Which I'm guessing is used as well for future adding to the menu.